src/components/RoomView/OperativeBoard.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import Card from './Card.jsx';
+import { useSelector } from 'react-redux';
+import { ref, update } from 'firebase/database';
+import { database } from '../../utils/firebase';
+import { Button } from '@mui/material';
+
+const OperativeBoard = () => {
+  const words = useSelector((state) => state.wordsInGame);
+  const roomId = useSelector((state) => state.player.roomId);
+  const playerId = useSelector((state) => state.player.playerId);
+  const teamOneRemainingCards = useSelector((state) => state.game.team1RemainingCards);
+  const teamTwoRemainingCards = useSelector((state) => state.game.team2RemainingCards);
+  const gameStatus = useSelector((state) => state.game.status);
+  const { teamOneOperatives } = useSelector((state) => state.teamOne);
+  const { teamTwoOperatives } = useSelector((state) => state.teamTwo);
+  let gameRef = ref(database, 'rooms/' + roomId + '/game/');
+
+  const teamOneOperativesIds = Object.values(teamOneOperatives).map((operative) => {
+    return operative.playerId;
+  });
+  const teamTwoOperativesIds = Object.values(teamTwoOperatives).map((operative) => {
+    return operative.playerId;
+  });
+
+  const style = {
+    display: 'grid',
+    gridTemplateColumns: 'auto auto auto auto auto',
+    marginTop: '5%',
+    gap: '2%',
+    justifyContent: 'center',
+    alightItems: 'center',
+  };
+
+  const endTurn = () => {
```

Comment:
maybe we can break this out into its own file bc its being used in multiple components, but ive had issues doing that in the past so maybe someone can help me with this :)